front_end/content.py

```diff
@@ -640,6 +640,7 @@ def get_assignment_statuses(self, course_id, user_id):
                  ORDER BY title'''
 
         assignment_statuses = []
+
```

Comment:
Thanks for adding vertical white space. I like to avoid having more than 5-6 consecutive lines of code without vertical white space because it's easier to digest mentally that way.